File: 1470-shuffle-the-array/1470-shuffle-the-array.py
class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        new_nums = [] # create an empty list
        for i in range(len(nums)//2): # loop through only half of the list
            new_nums.append(nums[i]) # append the first element
            new_nums.append(nums[i+n]) # append the [i+n]'th element
        return new_nums # returned the filled list

File: 1470-shuffle-the-array/1470-shuffle-the-array.java
class Solution {
    public int[] shuffle(int[] nums, int n) {
                //ans[] is used to store output data i.e.SC-O(1)
        int ans[] = new int[nums.length];
        int first = 0;
        int last = n;
        int i=0;
        while(first != n){
            ans[i++] = nums[first++];
            ans[i++] = nums[last++];
        }
        return ans;
    }
}